test/unit/mdc-drawer/dismissible.foundation.test.js

```diff
@@ -33,23 +32,6 @@ const setupTest = () => {
   return {foundation, mockAdapter};
 };
 
-test('exports strings', () => {
```

Comment:
Why were these 3 tests removed? They should exist for any foundation.

Reply:
I don't see any value in these tests. It is like testing the existence of class properties, there is no business logic involved. is it really needed?

Reply:
Given that I just cut a patch release yesterday with a fix related to us missing one of these, I would argue it is warranted :) see https://github.com/material-components/material-components-web/pull/3346

Reply:
Ah ok! Added these tests back.

Thanks!